tsconfig.configs.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "noEmit": true
+  },
+  "files": [
+    "./karma.conf.js",
+    "./gulpfile.js"
+  ]
+}
```

Comment:
What is the purpose of this tsconfig file?

Reply:
Since this is from #454, I'll answer there.

Reply:
Does not exist any more.